cli/utils/response.go

```diff
@@ -1,4 +1,4 @@
-package cli
+package utils
```

Comment:
`utils` is IMO the wrong name.

Everything in this package is for a client to the scheduler.